### DISCUSSION TITLE
computedWatchers

### DISCUSSION THREAD

**Discussion by zanzara** [quasarframework/quasar#10069] [General - Components / Directives / etc]
Sry, if I'm wrong here and my problem is only related to Vue3,
but I use this call  `this._computedWatchers.todaysDate.run();`  where `todaysDate` is a computed.

It worked all the the time in Q1 but now, migrating to Q2, I got an exception calling now this method. 

Any suggestions how to force to re-run a computed now within Q2/Vue3?

Edit: `_computedWatchers`  apparently doesn't exit anymore under Q2/Vue3

**Comment by metalsadman** [comment]
@zanzara you can use either `watchEffect` or `watch` of composition api https://v3.vuejs.org/api/computed-watch-api.html#watcheffect

**Reply by zanzara** [reply]
Thanks to @metalsadman  
But could you probably provide an example how to do it?
Pls. look as well here: [https://github.com/vuejs/vue/issues/214#issuecomment-692064376](https://github.com/vuejs/vue/issues/214#issuecomment-692064376)

I really got a little bit lost in my migration, with all those breaking changes that comes up with Vue3 and packages which depends, like (Vuex, Vue-router, I18n-v9, ...), even taking the whole quasar migrition guide into account.

I even got no succes in transforming into this [https://quasar.dev/options/app-internationalization#create-language-switcher](https://quasar.dev/options/app-internationalization#create-language-switcher) from the old approach.
In my case `const { locale } = useI18n({ useScope: 'global' })`  is empty i.e.

And why is this example here  [https://quasar.dev/options/quasar-language-packs#change-quasar-language-pack-at-runtime](https://quasar.dev/options/quasar-language-packs#change-quasar-language-pack-at-runtime) so different?

(I even could not imagine, to have a  `watch` wihtin `setup`  in Vue3!?  That's rather weired to me.
